.github/workflows/create-release-tag.yml

```diff
@@ -0,0 +1,110 @@
+name: Create Tag and Release with Changelog
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Tag name for the new release'
+        required: true
+
+permissions:
+  contents: write
+  packages: write
+  pull-requests: write
+  
+jobs:
+  fetch-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      old_tag: ${{ steps.get_tag.outputs.old_tag_name }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+          fetch-depth: 0
+      
+      - name: Get latest tag
+        id: get_tag
+        run: |
+          echo "old_tag_name=$(git ls-remote --tags origin | awk -F'/' '{print $3}' | grep -v '{}' | sort -V | tail -n1)" >> $GITHUB_OUTPUT
+      - name: print tag
+        id: print_tag
+        run: | 
+          echo "Old Tag=${{ steps.get_tag.outputs.old_tag_name }}"
+          echo "NEW_TAG=${{ github.event.inputs.tag_name }}" >> $GITHUB_ENV
+          echo "$(basename ${{ github.ref }})"
+
+      - name: Check if tag exists
+        id: check_tag
+        run: |
+         import sys
+         import subprocess
+         tag_name = "${{ github.event.inputs.tag_name }}"
+         command = ['git', 'tag', '-l', tag_name]
+         output = subprocess.check_output(command, stderr=subprocess.STDOUT)
+         if output.decode() != "":
+           print(f"Error: Tag '{tag_name}' already exists.", file=sys.stderr)
+           sys.exit(1)
+         else:
+           print(f"Tag '{tag_name}' does not exists.")
+        
+        shell: python
+        continue-on-error: false
+
+#this works only if params.env contains image:tag_version_number
+  update-params-env:
+    runs-on: ubuntu-latest
+    needs: fetch-tag
+    outputs:
+      param_env: ${{ steps.read_params_env.outputs.params_env }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+           
+      - name: Read params.env
+        id: read_params_env
+        run: |
+          echo "params_env=$(cat config/overlays/odh/params.env)" >> $GITHUB_ENV
+              
+      - name: Update params.env with new release version
+        run: |     
+          sed -i 's|:v[0-9.]*\b|:${{ github.event.inputs.tag_name }}|gm' config/overlays/odh/params.env
+      - name: Commit changes
+        run: |
+          git config --global user.email "github-actions@github.com"
+          git config --global user.name "GitHub Actions"
+          git add config/overlays/odh/params.env
+          git commit -m "Update image refs for odh release."
+          git push 
```

Comment:
I'd say we should not update the branch.

We have had people telling that the manifests on the release branches and also on the main branch, are pointing to quay tags that are not good for testing (devFlags in the operator).

So, I think a push to the branch should be omitted, and only the tag should be pushed.

Reply:
@israel-hdez But  as discussed with @selbi we need a commit message to be pushed with the updated values of params.env

Reply:
So, I think a push to the branch should be omitted, and only the tag should be pushed.

So you are trying to say we dont need to update the params.env file anymore with the tag values ??

because if we dont  commit how can we see the changes of the updated tag values in the params.env?

Reply:
> So you are trying to say we dont need to update the params.env file anymore with the tag values ??
> 
> because if we dont commit how can we see the changes of the updated tag values in the params.env?

We do need to update `params.env`, but we don't  need to push that commit to the `release-*` branch. Later, in the `Create Tag` step, you do `git push origin ${{ github.event.inputs.tag_name }}`. This is where you push the commit.

In short, I'm suggesting to only remove Line 81: `git push`.